packages/components/src/placeholder/README.md

```diff
@@ -11,3 +11,12 @@ const MyPlaceholder = () => (
 	/>
 );
 ```
+
+### Props
+
+Name | Type | Default | Description
+--- | --- | --- | ---
+`icon` | `string|element` | `undefined` | If provided, renders an icon next to the label.
```

Comment:
I'm not sure that it's totally clear that `element` means a React element. Maybe we could say `ReactElement`?